docker/Dockerfile.php

```diff
@@ -23,10 +44,23 @@
     mysqli \
     pdo \
     pdo_mysql \
-    zip \
+    zip
+
+# Build and install a newer version of Python. zkg requires at least
+# Python 3.9 so we can't update it without that.
+ENV PYENV_ROOT=/root/.pyenv
```

Comment:
I'm installing this is root's home directory just because that's usually the default. We can move it somewhere else if that's not ideal.